Gemfile

```diff
@@ -13,4 +13,5 @@ group :development, :test do
   gem "rspec-rails", "3.0.1"
   gem 'rspec-collection_matchers', '~> 1.0.0'
   gem "capybara", "2.3.0"
-end
+  end
+gem 'pry', '~> 0.10.1'
```

Comment:
You added `pry` everywhere instead of inside of the block for test and development. Lines 16 and 17 should be swapped.